redfish-core/lib/account_service.hpp

```diff
@@ -1349,6 +1349,11 @@ inline void updateUserProperties(
             // Check if the input vector contains just one bypass type:
             // as there are just two defined in the backend:
             // GoogleAuthenticator  and None
+            if (username == "service")
```

Comment:
can you please add service user checks to GenerateSecretKey and VerifyTOTP redfish operations??

Reply:
why? will not it be automatically covered by the bypass property? If bypass enabled generate secret key will not work right?

Reply:
what's error returned for generated secret key and verify totp operation for service ??

Reply:
But we should have same error for any user who are bypassed and yet try to create secret key right? Why special error for service user?

Reply:
then its going to return internal server error